src/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+addopts = --doctest-modules --junitxml=junit/test-results.xml --cov-report=xml --cov-report=term --cov --cov-report html --cov . --cov-config .coveragerc
+doctest_optionflags= IGNORE_EXCEPTION_DETAIL
```

Comment:
```suggestion
doctest_optionflags= IGNORE_EXCEPTION_DETAIL

```

Reply:
Do you mean I should delete these two lines?
```
**/tests/
doctest_optionflags= IGNORE_EXCEPTION_DETAIL
```
If we change these lines, `pytest.ini` and `.coveragerc` should move to the top-level directory.

Reply:
No, I mean there should be a newline at the end of each file. 😄

Reply:
And yes, I believe they should be at the top-level.